src/features/abi-methods/components/abi-value.tsx

```diff
@@ -14,7 +15,19 @@ export function AbiValue({ abiValue }: Props) {
     return <AbiArrayValue array={abiValue} />
   }
   if (abiValue.type === AbiType.String) {
-    return <span>{`"${abiValue.value}"`}</span>
+    return <span className="text-abi-string">{`"${abiValue.value}"`}</span>
+  }
+  if (abiValue.type === AbiType.Number) {
+    return <span className="text-abi-number">{`${abiValue.value}`}</span>
```

Comment:
```suggestion
    return <span className="text-abi-number">{abiValue.value}</span>
```
same for the bool value